x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/FieldExtractorTestCase.java

```diff
@@ -1766,6 +1753,7 @@ private Map<String, Object> runEsql(String query) throws IOException {
             request = request.pragmas(
                 Settings.builder().put(QueryPragmas.FIELD_EXTRACT_PREFERENCE.getKey(), preference.toString()).build()
             );
+            request.pragmasOk();
```

Comment:
I think we need to guard this for 8.16+ because accept_pragma_risks is only available in 8.16+, and FieldExtractorTestCase can run in a mixed cluster.